flutter/android/android.mk

```diff
@@ -33,7 +33,7 @@ else
 endif
 
 .PHONY: flutter/android/libs/build
-flutter/android/libs/build:
+flutter/android/libs/build: flutter/libqtibackend/patchlibs
```

Comment:
Should `patchelf` be used in the master branch?

Reply:
I think we can merge it for now, and with android ndk 25, we can remove it.